app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckExtractionService.scala

```diff
@@ -53,8 +58,15 @@ class HecTaxCheckExtractionServiceImpl @Inject() (
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
   val maxTaxChecksPerFile: Int   = config.get[Int]("hec-tax-heck-file.default-size")
-  val hec: String                = "HEC"
-  val sdesDirectory: String      = "sdes"
+
+  val informationType: String           = config.get[String]("file-notify-config.information-type")
+  val recipientOrSender: String         = config.get[String]("file-notify-config.recipient-or-sender")
+  val baseLocation: String              = config.get[String]("file-notify-config.location")
+  val algorithm: String                 = config.get[String]("file-notify-config.checkSum.algortihm")
```

Comment:
we may not need this to be read from config - we wouldn't be able to switch to another algorithm by changing the config cos it's hard coded to give an MD5 value anyways